src/app/index.module.ts

```diff
@@ -219,12 +247,12 @@ angular.element(document).ready(() => {
     cheBranding.ready.then(() => {
       (angular as any).resumeBootstrap();
     });
-  }).catch((error: string) => {
-    console.error(`Can't GET "/api". ${error ? 'Error: ' : ''}`, error);
-    if (!hasCertificateError) {
-      error = `${error}<div>Click <a href="/">here</a> to reload page.</div>`
+  }).catch((errorMessage: string) => {
+    if (hasSSO) {
+      showErrorMessage(errorMessage);
+    } else {
+      showErrorMessage(`${errorMessage}<div>Click <a href="/">here</a> to reload page.</div>`);
```

Comment:
Why do we have a difference in this case?
```
...<div>Click <a href="/">here</a> to reload page.</div>
```

Reply:
I slightly refactored the code you'd added to handle errors related to certificate issues. I assume that's because in case of issue related to SSO/certificate user need to somehow resolve the reason of the issue, in case of other issues simple page refreshing may help.